src/ui/profile_preferences.ui

```diff
@@ -3775,5 +3790,22 @@ you can use it but there could be issues with aligning columns of text</string>
  <resources>
   <include location="../mudlet.qrc"/>
  </resources>
- <connections/>
+ <connections>
+  <connection>
+   <sender>enableDarkTheme</sender>
+   <signal>toggled(bool)</signal>
+   <receiver>label_darkEditorPrompt</receiver>
+   <slot>setVisible(bool)</slot>
+   <hints>
+    <hint type="sourcelabel">
+     <x>81</x>
+     <y>278</y>
+    </hint>
+    <hint type="destinationlabel">
+     <x>82</x>
+     <y>305</y>
+    </hint>
+   </hints>
+  </connection>
+ </connections>
```

Comment:
This is the new bit.

Reply:
Though that is also possible to do in the `dlgProfilePreferences` class - and at least if it is done *there* it will not be hidden away where coders may not spot it...

:man_shrugging: